src/pipelines/skeleton.ml

```diff
@@ -89,3 +89,50 @@ let v ~build ~config ~platform ~mirage ~repos mirage_skeleton =
       let skeleton = Current.map Current_git.Commit.id skeleton in
       { mirage; repos; skeleton; build })
     mirage_skeleton
+
+(* Test all of mirage-skeleton at once *)
+let all_in_one_test ~(platform : Platform.t) ~target ~repos ~config
+    mirage_skeleton =
+  let spec =
+    let+ repos = repos in
+    let open Obuilder_spec in
+    Platform.spec platform.system
+    |> Spec.add (Setup.add_repositories repos)
```

Comment:
we don't want to `opam add` opam-overlay here. We should test the exact same experience as a user wanting to compiler `mirage-skeleton`: e.g. opam add `mirage-dev` and that's it.